serde/src/de/content.rs

```diff
@@ -0,0 +1,732 @@
+use core::fmt;
```

Comment:
I added a comment in 066c9a15cfc0a37111ae615ce57b44f7de3b6b1e and filed #741 to follow up on making it public.